conan/api/model.py

```diff
@@ -9,14 +9,18 @@
 from conans.util.files import load
 from conans.model.version_range import VersionRange
 
+OSS_RECIPES = "oss-recipes"
+OSS_RECIPES_GIT = "oss-recipes-git"
```

Comment:
One detail to check: for both the `oss-recipes` and the `oss-recipes-git` with an ssl url I get a warning about the format of the URL like `WARN: The URL 'git@github.com:conan-io/conan-center-index.git' is invalid. It must contain scheme and hostname`.

Reply:
Yes, that's right, I need to get rid of it

Reply:
Removed warning from ``conan remote add``. The warning will remain in ``conan remote update``, but that one can wait for a refactor of ``RemoteRegistry``